dataprofiler/profilers/profiler_options.py

```diff
@@ -143,16 +143,21 @@ def _validate_helper(self, variable_path='BooleanOption'):
 
 class HistogramOption(BooleanOption):
 
-    def __init__(self, method=['auto']):
+    def __init__(self, is_enabled=True, method=None, hist_bin_count=None):
```

Comment:
i think it should be more clear if method is defaulted as 'auto', as we implied the whole list of methods if it is None.

Reply:
The reason I chose this route is that if someone were to write:
```python
HistogramOptions(hist_bin_count=100)
```
How do you know what to choose? Maybe the user specified `'auto'` as well? 
Maybe they wrote `HistogramOptions(method='auto', hist_bin_count=100)`
Ultimately, we need to protect against both from being set simultaneously.
If we revert it, the user would then have to write:
```python
HistogramOptions(method=None, hist_bin_count=100)
```

Unless we want to always say if `hist_bin_count` is specified to  choose `hist_bin_count`. But to not get a warning, a user will have to write as above.

I'm open to suggestions.

Reply:
Actually, if we combine `method` with `hist_bin_count` to have only something like `bins`, that'd make the most sense. So, it can be str, list(str), int, or None.
https://numpy.org/doc/stable/reference/generated/numpy.histogram.html